packages/keyring/src/pair/index.ts

```diff
@@ -62,6 +62,10 @@ function verify (type: KeypairType, message: Uint8Array, signature: Uint8Array,
     : naclVerify(message, signature, publicKey);
 }
 
+function isLocked (secretKey?: Uint8Array): boolean {
```

Comment:
```suggestion
function isLocked (secretKey?: Uint8Array): secretKey is Uint8Array {
```

If passing, it will be seen as a valid u8a

Reply:
Done. Good suggestion